Description: <default session>
 Change default session from gnome to default in daemon for blankon.
 .
 gdm3 (3.26.2.1-2blankon1) uluwatu; urgency=medium
 .
   * New upstream release
   * Change default session
Author: AnTo SamalonA <ansamsoftdev@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2017-11-13

--- gdm3-3.26.2.1.orig/daemon/gdm-session.c
+++ gdm3-3.26.2.1/daemon/gdm-session.c
@@ -541,7 +541,7 @@ get_fallback_session_name (GdmSession *s
                 }
         }
 
-        name = g_strdup ("gnome");
+        name = g_strdup ("default");
         if (get_session_command_for_name (self, name, NULL)) {
                 g_free (self->priv->fallback_session_name);
                 self->priv->fallback_session_name = name;
